### PR TITLE
Fixed user activitiy usages showing all alignments as rubric alignmen…

### PR DESCRIPTION
**Pull request by mpharoah-d2l**
…ts and being unassessable

### CODE REVIEW

**Comment by ChrisKraljevicD2L** [comment]
Is there a reason CI isn't running on this branch?

**Comment by mpharoah-d2l** [comment]
Because GitHub sucks

**Comment by ChrisKraljevicD2L** [comment]
Also, does this mean that any uses of `activity-alignments` that are being used for assessing outcomes need to now use hrefs for user activity usages instead of activity usages?

**Comment by mpharoah-d2l** [comment]
They already were. Now I'm just handlingh them different to prevent them from being interpreted as rubric alignments

File: d2l-user-alignment-list.js [hunk 1]
```diff
@@ -0,0 +1,208 @@
...
+
+			d2l-loading-spinner {
+				width: 100%;
+				margin: var(--d2l-alignment-list-overflow-margin);
+			}
+
+			siren-entity-loading {
+				overflow: hidden;
+				width: 100%;
+				margin: -var(--d2l-alignment-list-overflow-margin);
```
Comment (jstefaniuk-d2l) [inline d2l-user-alignment-list.js]: typo?

**Comment by mpharoah-d2l** [comment]
CircleCI is RIP

File: d2l-user-alignment-list.js [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+			}
+
+			siren-entity-loading d2l-loading-spinner {
+				--d2l-loading-spinner-size: 10rem;
+			}
+
+			ul {
+				padding: 0;
+				overflow: auto;
+				margin: 0;
```
Comment (jstefaniuk-d2l) [inline d2l-user-alignment-list.js]: can be done with shorthand
`margin: 0.9rem 0 0 0;`

File: d2l-user-alignment-list.js (hunk 1 above)
Comment (mpharoah-d2l) [reply d2l-user-alignment-list.js]: No, this is just copied from the d2l-alignment-list. It just ended up being easier to copy the file and modify it than to make d2l-alignment-list do both things, but I still want the same styling in both

Comment (jstefaniuk-d2l) [reply d2l-user-alignment-list.js]: what's `-var`? I've never seen it before and can't google it due to google not being good for searching the `-` character

Comment (mpharoah-d2l) [reply d2l-user-alignment-list.js]: It lets you use variable css in web components. `d2l-icon` uses it a lot, so you can look there for exzmples

**Comment by mpharoah-d2l** [comment]
CI machine has run amok, but it works on my local BSI
image

**Comment by ChrisKraljevicD2L** [comment]
I hope yer local BSI missed my changes because the LoAs shouldn't be side by side like that in assignments :/

**Comment by mpharoah-d2l** [comment]
I don't have your changes in my branch

**Comment by mpharoah-d2l** [comment]
Trying it with your changes now, but local BSI takes forever and a half to build

**Comment by ChrisKraljevicD2L** [comment]
Oh don't worry about that, I can test my stuff locally instead.

File: d2l-user-alignment-list.js (hunk 1 above)
Comment (jstefaniuk-d2l) [reply d2l-user-alignment-list.js]: https://github.com/search?q=org%3ABrightspaceUI+-var&type=Code not seeing it

**Comment by mpharoah-d2l** [comment]
image

File: d2l-user-alignment-list.js (hunk 1 above)
Comment (mpharoah-d2l) [reply d2l-user-alignment-list.js]: Pretty sure that searches for everything not containing `var` :P

Comment (jstefaniuk-d2l) [reply d2l-user-alignment-list.js]: Seems like the only instance of -var for css in the whole org is that activity alignments file: https://github.com/search?p=3&q=org%3ABrightspace+-var&type=Code

Comment (mpharoah-d2l) [reply d2l-user-alignment-list.js]: Oh, the hyphen. I thought you just meant using `var( --something )`
Yeah, that's a mistake lol

Comment (jstefaniuk-d2l) [reply d2l-user-alignment-list.js]: Lol I could have been a bit more clear

**Comment by mpharoah-d2l** [comment]
@jstefaniuk-d2l were you wanting to review this as well, or shall I merge?

**Comment by jstefaniuk-d2l** [comment]
Merge away!

**Pull request merged**